README.md

```diff
@@ -14,183 +14,315 @@
+  beta2: 0.95
+
+  #   (type: Optional[float], default: null)
+  max_norm:
+
+  #   (type: float, default: 6e-05)
+  min_lr: 6.0e-05
+
+# Evaluation-related arguments. See ``litgpt.args.EvalArgs`` for details
+eval:
+
+  # Number of optimizer steps between evaluation calls (type: int, default: 100)
+  interval: 100
+
+  # Number of tokens to generate (type: Optional[int], default: 100)
+  max_new_tokens: 100
+
+  # Number of iterations (type: int, default: 100)
+  max_iters: 100
+
+# The name of the logger to send metrics to. (type: Literal['wandb', 'tensorboard', 'csv'], default: csv)
+logger_name: csv
+
+# The random seed to use for reproducibility. (type: int, default: 1337)
+seed: 1337
+```
 
-- [Finetune with Adapters](tutorials/finetune_adapter.md)
-- [Finetune with LoRA or QLoRA](tutorials/finetune_lora.md)
 
 &nbsp;
 
-### Understanding finetuning -- conceptual tutorials
+## LitGPT design principles
 
-Looking for conceptual tutorials and explanations? We have some additional articles below:
+This repository follows the main principle of **openness through clarity**.
 
-- [Understanding Parameter-Efficient Finetuning of Large Language Models: From Prefix Tuning to LLaMA-Adapters](https://lightning.ai/pages/community/article/understanding-llama-adapters/)
+**LitGPT** is:
+
+- **Simple:** Single-file implementation without boilerplate.
+- **Correct:** Numerically equivalent to the original model.
+- **Optimized:** Runs fast on consumer hardware or at scale.
+- **Open-source:** No strings attached.
 
-- [Parameter-Efficient LLM Finetuning With Low-Rank Adaptation (LoRA)](https://lightning.ai/pages/community/tutorial/lora-llm/)
+Avoiding code duplication is **not** a goal. **Readability** and **hackability** are.
 
 &nbsp;
 
-## Pretraining
+## Get involved!
 
-We provide simple training scripts based on Fabric if you want to venture into pretraining. Conversion scripts for our optimized streaming `PackedDataset` are included.
+We appreciate your feedback and contributions. If you have feature requests, questions, or want to contribute code or config files, please don't hesitate to use the [GitHub Issue](https://github.com/Lightning-AI/litgpt/issues) tracker.
 
-Follow this guide to start pretraining on
+We welcome all individual contributors, regardless of their level of experience or hardware. Your contributions are valuable, and we are excited to see what you can accomplish in this collaborative and supportive environment.
 
-- [RedPajama, a reproduction of LLaMA's training set](tutorials/pretrain_redpajama.md)
-- [OpenWeb Text, a reproduction of GPT-2's dataset](tutorials/pretrain_openwebtext.md)
+&nbsp;
 
+> [!TIP] 
+> Unsure about contributing? Check out our [How to Contribute to LitGPT](https://lightning.ai/pages/community/tutorial/how-to-contribute-to-litgpt/) guide.
+
+If you have general questions about building with LitGPT, please [join our Discord](https://discord.gg/VptPCZkGNa).
 
 
 &nbsp;
 
+## Tutorials and how-to guides
```

Comment:
Can we make the `tutorials/` directly more prominent? Also include the word "docs" for those who cmd+F it